packages/CollapsibleChecklists/src/CollapsibleChecklists.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Heading from "./components/Heading";
+import Group from "./components/Group";
+import Item from "./components/Item";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  onChange: PropTypes.func.isRequired,
+};
+
+const defaultProps = {};
+
+const CollapsibleChecklists = props => {
+  const { children, onChange } = props;
+  const modifiedChildren = [];
+
+  React.Children.forEach(children, (child, index) => {
+    switch (child.type.displayName) {
+      case Group.displayName:
+        modifiedChildren.push(React.cloneElement(child, { key: `group${index}`, onChange })); // eslint-disable-line react/no-array-index-key
```

Comment:
That ESLint rule speaks the truth. Even if you're not explicitly reordering items a la `<Sortable>`, when an item gets removed from the middle of the list things will break. https://reactjs.org/docs/lists-and-keys.html#keys